Solutions/Endjin.Imm.Domain/Processing/NullRuleCalculator.cs

```diff
@@ -5,12 +5,12 @@
 
     public class NullRuleCalculator : IRuleCalculator
     {
-        public decimal Percentage(Rule rule, IEvaluationContext context)
+        public decimal Percentage(RuleAssertion rule, IEvaluationContext context)
```

Comment:
Should the parameter name (and I suspect the interface definition) be updated too?

Reply:
Good point. Fixed.